### DISCUSSION TITLE
Is there a better way to get assistance?

### DISCUSSION THREAD

**Discussion by rmjwilbur** [artilleryio/artillery#1927] [Artillery]
Hi there, 

I've run across several issues that are dated, open, and not responded to. I've been encountering some of the same issues. Is there a slack or discord channel that people are using instead of the issues and discussions?

Thanks in advance

**Comment by hassy** [comment]
hi @rmjwilbur 👋  apologies, it's too easy to miss new comments on older issues.

I replied in https://github.com/artilleryio/artillery/issues/189. Please feel free to link any other issues as well!

**Comment by rmjwilbur** [comment]
Thanks @hassy - greatly appreciated - between your answer and my comment, I think https://github.com/artilleryio/artillery/issues/892 is resolved as well
